template-contributing-guide.md

```diff
@@ -0,0 +1,341 @@
+        </ul>
+    </td>
+    <td><ul>
+          <li>Template mentor</li>
+          <li>Templates coordinator</li>
+        </ul>
+    </td>
+  </tr>
+</table>
+
+Each phase is explained in more depth in the remaining sections.
+
+
+## Join the community
+
+To become a full-fledged templateer, you’ll want to join our communication channels so that you can talk to us:
+
+- **Slack** - Our [Slack workspace](https://thegooddocs.slack.com/) is one of the primary means of communicating with members of our project. After joining our workspace, join the `#welcome` channel to introduce yourself to the welcoming committee. Consider also joining these Slack channels if you plan to work on a template:
+  - `#templates`
+  - `#community-docs`
+  - `#welcome`
+  - `#topical`
+- **Group forums** - Join our [Groups.io mailing list](https://thegooddocsproject.groups.io/g/main/topics) to communicate with other templateers over email. These messages can be grouped into a daily digest for your convenience.
+- **Working groups** - Our project is organized into several different working groups that meet on a regular basis to work on the project’s key initiatives. One of the best ways to get started with our project is to join and meet with one of our working groups. See [The Good Docs Project Working Groups](https://thegooddocsproject.dev/working-group/) for a list of our current active groups. NOTE: If you plan to contribute to our project by writing templates, you will be required to join one of the template-focused working groups.
+- **Weekly meetings** - The project leaders hold two weekly meetings to discuss project-level decisions. Feel free to join one of these meetings to introduce yourself to the project leaders and discover next steps for getting involved in the project. We currently hold meetings for the U.S./Canada and APEC/EU/Africa time zones. After introducing yourself on Slack, one of the members of the welcoming committee can help you get invited to the best meeting for your time zone.
+
+As you begin to join our project, remember that this is a project composed entirely of volunteers.
+We love to welcome new members, but want to be careful not to burn out our core project contributors.
+This level of mindfulness helps us ensure that we retain our project’s capacity to produce high-quality work.
+As such, we ask that you respect the time of our project maintainers and contributors.
+(And expect us to respect your time in return!)
+
+We expect all members of our project to be nice to each other and to follow our [Code of Conduct](https://thegooddocsproject.dev/code-of-conduct/) when interacting with other members of the Good Docs Project.
+
+
+## Adopt a template
+
+In this phase, you will decide which template project you will work on and adopt that template.
+
+Be aware that:
+- Each template project is represented by a corresponding issue in the [Good Docs Project templates](https://github.com/thegooddocsproject/templates/issues) repository.

```

Comment:
Good place to mention the `templates` label.

Reply:
We'll have to revise this document after the fact as our project management tools and processes change.